lib/duplicate_deleter.rb

```diff
@@ -62,8 +62,13 @@ def call(ids, id_type: 'content_id')
 
       content_ids = results[:results].map { |a| a['content_id'] }
       if content_ids.uniq.count != 1
-        io.puts "Skipping #{id_type} #{id} as multiple content_id's detected #{content_ids.uniq.join(', ')}"
-        next
+        if results[:results].any? { |a| a[:elasticsearch_type] != type_to_delete && a['content_id'].nil? }
+          io.puts "Skipping #{id_type} #{id} as it is indexed with a valid '_type' but a missing content ID"
```

Comment:
I think this message is a bit unclear about which result is missing the content ID.

Maybe something like
```
"Skipping #{id_type} #{id} as there is another document indexed with a valid '_type' but a missing content ID"
```